src/lucky/redirectable.cr

```diff
@@ -1,19 +1,68 @@
+# Redirect the request
+#
+# There are multiple ways to redirect inside of an action. The most common ways are to redirect to a `Lucky::Action` class, or a URL/path `String`. Both use the `redirect` method:
+#
+# ```crystal
+# redirect to: Users::Index
+# redirect to: Users::Show.with(user.id)
+# redirect to: "https://luckyframework.org/"
+# redirect to: "/users"
+# ```
+#
+# By default, the method will set the status code to `302` A.K.A. "Found". If you want to customize the status code, you can pass it directly:
+#
+# ```crystal
+# redirect to: Users::Index, status: 301
+#
+# # or use the built in enum value
+# redirect to: Users::Index, status: Status::MovedPermanently
+# ```
+#
+# You can find a list of all of the possible statuses [here](https://github.com/luckyframework/lucky/blob/master/src/lucky/action.cr).
+#
+# Internally, all the different methods in this module eventually use the
+# method that takes a `String`. However, it's recommended you pass a
+# `Lucky::Action` class if possible because it guarantees runtime safety.
 module Lucky::Redirectable
+  # Redirect using a `Lucky::RouteHelper`
+  #
+  # ```crystal
+  # route = Lucky::RouteHelper.new(method: :get, path: "/users/1")
+  # redirect to: route, status: 301
+  # ```
   def redirect(to route : Lucky::RouteHelper, status = 302)
     redirect to: route.path, status: status
   end
 
+  # Redirect to a `Lucky::Action`
+  #
+  # ```crystal
+  # redirect to: Users::Index
+  # redirect to: Users::Show.with(user.id), status: 301
```

Comment:
This line wouldn't work for this because it would generate a RouteHelper

Reply:
Oh I see, yeah. I guess it's not possible to pass parameters with an action class. It would need a third argument.